src/Altinn.App.Core/Internal/Process/Elements/AltinnExtensionProperties/AltinnPaymentConfiguration.cs

```diff
@@ -12,5 +14,45 @@ public class AltinnPaymentConfiguration
         /// </summary>
         [XmlElement("paymentDataType", Namespace = "http://altinn.no/process")]
         public string? PaymentDataType { get; set; }
+
+        internal ValidAltinnPaymentConfiguration Validate()
+        {
+            List<string>? errorMessages = null;
```

Comment:
Honestly don't really need this list of error message anymore. Had more config values before, hence the list, but now that we only check one value it's not really needed. MIGHT be handy in the future. Up to you if you want to simply it.

Reply:
Ahh I see. Since it was a bit of a hassle convincing the compiler of the nullability flow, I'll keep it as is for now so I remember this works 😄